manager/assets/modext/widgets/fc/modx.panel.fcset.js

```diff
@@ -167,9 +163,7 @@ Ext.extend(MODx.panel.FCSet,MODx.FormPanel,{
 
     ,setup: function() {
         if (!this.initialized) {this.getForm().setValues(this.config.record);}
-        if (!Ext.isEmpty(this.config.record.controller)) {
-            Ext.getCmp('modx-fcs-header').update(_('set')+': '+this.config.record.controller);
-        }
+        Ext.getCmp('modx-fcs-header').html = Ext.util.Format.htmlEncode(_('set'));
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(_('set'));`